MBBSEmu.Tests/CPU/GetOperandOffset_Tests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Iced.Intel;
+using System;
+using Xunit;
+using static Iced.Intel.AssemblerRegisters;
+
+namespace MBBSEmu.Tests.CPU
+{
+    public class GetOperandOffset_Tests : CpuTestBase
+    {
+
+        [Theory]
+        [InlineData(0, 1, 1, 0, 1)]
+        [InlineData(1, 1, 2, 0, 0)]
+        [InlineData(0xFFFF, 1, 0, 1, 0)]
+        [InlineData(0xFFFE, 1, 0xFFFF, 1, 1)]
```

Comment:
what happens if [BX + DI] overflows 65535, does the processor access the lower value or does it invoke an Overflow interrupt?

Reply:
i'm not sure -- but that'd be an error in the compiler if that case were to happen as no segment in an NE format DLL/EXE is > 0xFFFF bytes.

Reply:
I'm pretty sure MajorMUD relies on overflowing memory references as a copy protection mechanism.

Just kidding.